llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -26161,6 +26165,8 @@ SDValue DAGCombiner::visitFP_TO_BF16(SDNode *N) {
   return SDValue();
 }
 
+SDValue DAGCombiner::visitBF16_TO_FP(SDNode *N) { return visitFP16_TO_FP(N); }
```

Comment:
You should probably add an assert to the start of visitFP16_TO_FP to check that the opcode is BF16_TO_FP or FP16_TO_FP